src/bundle-configuration/zookeeper-prod/runtime-config.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export ZK_DATA_DIR=/var/lib/zookeeper
```

Comment:
Also I'd like to point up our ES has `/var/lib/elasticsearch` and `/var/log/elasticsearch` for data and logs to match default ES `1.5`'s default directories - search for `The location of the data files` and `Log files location`:

https://www.elastic.co/guide/en/elasticsearch/reference/1.5/setup-dir-layout.html